### ISSUE TITLE
Tried 4.5 bios version on 2 PSone, able to restore memory card afterwards with PS2 and PS1 itself.

### ISSUE THREAD

**Issue by Ichisuke83** [brad-lin/FreePSXBoot#18]
Just reporting that it is working fine on 2 PSOne consoles I've tried. Both with 4.5 version. With 4.4 wasn't working.
Probably both consoles have latest bios revision. Tried with an old third party memory card flashed with PS2 Memory Card Annihilator v2. 
What I noticed is that I was able to reflash the memory card from 4.4 to 4.5 always using Memory Card Annihilator without it stopping from recognising the memory card and freezing.
But then I flashed the memory card with the "bios 3.x" version to try on an SCPH-1002 PS1 and then I wasn't able to use Memory Card Annihilator anymore... Until I tried Memory Card Manager (MCM.EXE for PS1) and I was able to format the memory card and get it working normally again!
You can find MCM on PAV_PSXHKB7 cd image collection to use on the console. I'll attach here only the MCM.exe
[MCM.zip](https://github.com/brad-lin/FreePSXBoot/files/6305325/MCM.zip)
Note: MCM won't load with the modified memory card inserted. Let the program load and then insert the memory card to format with the PS1. Tried this on a real PS1 model SCPH-1002.

**Comment by whitezombie2000** [comment]
Hello! How do you use MCM.EXE on the PS1? I just tested the PSone freepsxboot-unirom-20210412-101 image and it worked fine. I flashed freepsxboot-unirom-20210412-5001-5501-5503-7003 image, couldn't get it work on a NTSC 5501, and now have the same issue. MCKiller and MCAnnihilator freeze as soon as I insert it on the PS2 for formatting or load a new image.

**Comment by whitezombie2000** [comment]
I think I might know now. I am will try the PSX 4in1 Ultimate Cheat Collection and see if I can format it.

**Comment by whitezombie2000** [comment]
Ok I was able to reformat the card by using PSX Hacker Kit (18 in 1) and loading Memory Card Manager. Whew...I thought my MC Card was toast!

**Comment by Ichisuke83** [comment]
> Ok I was able to reformat the card by using PSX Hacker Kit (18 in 1) and loading Memory Card Manager. Whew...I thought my MC Card was toast!

Nice you figured it out. Yeah I didn't post a link to the disc image itself of that collection since I think it contains copyrighted material.

**Comment by whitezombie2000** [comment]
I just wanted to mention that I get some inconsistencies with models. I have six NTSC SCPH-5501 models and five of them will boot this hack on a regular basis. One SCPH-5501 just refuses to do anything when clicking memory card. I have tested four PSone one models with a PM-41 motherboard. These have the large circular rubber pads on the bottom. Two of them will boot the hack but the other two just freeze when selecting memory card. I also tested four PSone models with a PM-41 (2) motherboard. These have the small circular rubber pads on the bottom. All four of these will boot the hack. Just wanted to report my findings.

**Comment by Ichisuke83** [comment]
> I just wanted to mention that I get some inconsistencies with models. I have six NTSC SCPH-5501 models and five of them will boot this hack on a regular basis. One SCPH-5501 just refuses to do anything when clicking memory card. I have tested four PSone models with a PM-41 motherboard. These have the large circular rubber pads on the bottom. Two of them will boot the hack but the other two just freeze when selecting memory card. I also tested four PSone models with a PM-41 (2) motherboard. These have the small circular rubber pads on the bottom. All four of these will boot the hack. Just wanted to report my findings.

Do the 2 PSone not booting the exploit have the "colored" menu screen like the latest fat models? Or maybe they have a modchip. Another person wrote that it doesn't work on its modchipped PSone...

**Comment by whitezombie2000** [comment]
Here are my findings. The PSone models all have the gray memory card/CD Player screen.

These are all NTSC models.

PM-41 (2) with MM3 modchip works
PM-41 (2) working no mods
PM-41 (2) working no mods
PM-41 not working freezes no mods
PM-41 not working freezes no mods
PM-41 working no mods
PM-41 working MM3 modchip installed
SCPH-5501 working no mods
SCPH-5501 working no mods
SCPH-5501 working no mods
SCPH-5501 working X-Station installed
SCPH-5501 working PSIO switchboard installed, PSIO not attched.
SCPH-5501 not working freezes no mods

**Comment by hurda** [comment]
PU-18-boards (5xxx) had 8 different (minor) revisions (-11 to -81), PM41* (10x) at least the same (-11 to -71 and -82).
Maybe those that don't work have something in common (other board-model, -revision, BIOS for some odd reason)?

For the non-working PM-41, maybe it's one of the [early ones with 4.4 BIOS](https://github.com/rarestMeow/playstation_models_bios_guide)?

**Comment by whitezombie2000** [comment]
Ok, I loaded PSX BIOS dumper 2.6 on some these to get the BIOS info. You are correct hurda. Two of the PM-41 have the 4.4 BIOS. I flashed the freepsxboot-unirom-20210412-102_4.4 image to a memory card and it works on the two SCPH-101 PM-41 models with the 4.4 BIOS. I still can't get the one SCPH-5501 to load the exploit and it has the same BIOS as the working SCPH-5501 models. The next task may to disassemble some of them and look at the board revision.

These are all NTSC models.

PM-41 (2) with MM3 modchip works
PM-41 (2) working no mods BIOS CRC-32 8d8cb7e4 Bios Version: 3.0 Date 11/18/96
PM-41 (2) working no mods BIOS CRC-32 8d8cb7e4 Bios Version: 3.0 Date 11/18/96
PM-41 not working freezes no mods BIOS CRC-32 6a0e22a0 Bios Version: 4.4 Date 3/24/00
PM-41 not working freezes no mods BIOS CRC-32 6a0e22a0 Bios Version: 4.4 Date 3/24/00
PM-41 working no mods BIOS CRC-32 171bdcec Bios Version: 4.5 Date 5/25/00
PM-41 working MM3 modchip installed BIOS CRC-32 171bdcec Bios Version: 4.5 Date 5/25/00
SCPH-5501 working no mods BIOS CRC-32 8d8cb7e4 Bios Version: 3.0 Date 11/18/96
SCPH-5501 working no mods BIOS CRC-32 8d8cb7e4 Bios Version: 3.0 Date 11/18/96
SCPH-5501 working no mods
SCPH-5501 working X-Station installed
SCPH-5501 working PSIO switchboard installed, PSIO not attched.
SCPH-5501 not working freezes no mods Bios CRC-32 8d8cb7e4 BIOS Version: 3.0 Date 11/18/96
